[2024-06-18T14:35:49.124+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T14:35:49.138+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fourthddag.task3 manual__2024-06-18T14:35:42.513031+00:00 [queued]>
[2024-06-18T14:35:49.142+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fourthddag.task3 manual__2024-06-18T14:35:42.513031+00:00 [queued]>
[2024-06-18T14:35:49.142+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-18T14:35:49.150+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task3> on 2024-06-18 14:35:42.513031+00:00
[2024-06-18T14:35:49.154+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1189) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T14:35:49.155+0000] {standard_task_runner.py:63} INFO - Started process 1191 to run task
[2024-06-18T14:35:49.155+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fourthddag', 'task3', 'manual__2024-06-18T14:35:42.513031+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/fourthdag.py', '--cfg-path', '/tmp/tmpk1mj4o6s']
[2024-06-18T14:35:49.156+0000] {standard_task_runner.py:91} INFO - Job 34: Subtask task3
[2024-06-18T14:35:49.186+0000] {task_command.py:426} INFO - Running <TaskInstance: fourthddag.task3 manual__2024-06-18T14:35:42.513031+00:00 [running]> on host a6ae00f47896
[2024-06-18T14:35:49.246+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fourthddag' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T14:35:42.513031+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T14:35:42.513031+00:00'
[2024-06-18T14:35:49.246+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T14:35:49.254+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T14:35:49.254+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2024-06-18T14:35:49.262+0000] {subprocess.py:86} INFO - Output:
[2024-06-18T14:35:54.264+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T14:35:54.266+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T14:35:54.309+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fourthddag, task_id=task3, run_id=manual__2024-06-18T14:35:42.513031+00:00, execution_date=20240618T143542, start_date=20240618T143549, end_date=20240618T143554
[2024-06-18T14:35:54.348+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T14:35:54.375+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T14:35:54.378+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
